### PR TITLE
[keymgr] Widen sw_binding and salt registers

### PR DESCRIPTION
**Pull request by tjaychen**
Address #5194

Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

**Review by weicaiyang — APPROVED** [review]
LGTM

File: hw/ip/keymgr/data/keymgr.hjson [hunk 1]
```diff
@@ -175,9 +175,14 @@
       randcount: "256",
       randtype:  "data",
     },
-
-    { name: "NumInReg",
-      desc: "Number of Registers for SW inputs (SW binding / keyID / Salt)",
+    { name: "NumSaltReg",
+      desc: "Number of Registers for SW inputs (Salt)",
+      type: "int",
+      default: "4",
```
Comment (weicaiyang) [inline hw/ip/keymgr/data/keymgr.hjson:181]: Sorry, I have a question
from this issue #5194, @alphan said "We decided to use 256b for sw_binding and salt"
should this be 8? 8x32=256.

Comment (tjaychen) [reply hw/ip/keymgr/data/keymgr.hjson:181]: ah crap. you're right. I separated out the parameters and forgot to change it to 8...

**Comment by alphan** [comment]
Reviewed the new PR, apologies for the delay. Thanks for catching this @weicaiyang!

**Pull request merged**